.circleci/config.yml

```diff
@@ -0,0 +1,76 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/coala
+
+    docker:
+      - image: circleci/python:3.6.1
```

Comment:
Your Circle 2.0 build has two jobs both running 3.6.1, one of which is completely redundant. 

On circle CI 1.0, we have two jobs, running two different versions of python, and neither of them are 3.6.1.

Please replicate the old behaviour.